public/index.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
```

Comment:
shouldn't we use strict types in this class too?

Reply:
Will solve it in the next PR! 🎉